crates/bevy_render/src/camera/projection.rs

```diff
@@ -138,7 +138,15 @@ impl Default for PerspectiveProjection {
 #[derive(Debug, Clone, Reflect, FromReflect, Serialize, Deserialize)]
 #[reflect(Serialize, Deserialize)]
 pub enum WindowOrigin {
+    /// Scale the view frustum from the center.
```

Comment:
Wow, thank you for looking into this! This is so much clearer.

Can you clarify the following? 

```
    /// When resizing the window, object in the center of the screen remain in the center, while opposite
    /// faces of the view frustum expand/shrink equally.
``` 

I understand, that in contrast to `BottomLeft`, which keeps the bottom and left faces fixed, this will change all the faces equally to perform the scaling operation. But I am not sure I understand the significance of "objects in the center of the screen remain in the center".

What happens to objects at the center of the screen when using `BottomLeft`?